src/transaction/log_recovery_redo_parallel.cpp

```diff
@@ -521,28 +517,21 @@ namespace cublog
 
   constexpr unsigned short redo_parallel::redo_task::WAIT_AND_CHECK_MILLIS;
 
-  redo_parallel::redo_task::redo_task (std::size_t a_task_idx
-				       , redo_parallel::task_active_state_bookkeeping &task_state_bookkeeping
-				       , redo_parallel::redo_job_queue &a_queue)
+  redo_parallel::redo_task::redo_task (std::size_t a_task_idx,
+				       redo_parallel::task_active_state_bookkeeping &task_state_bookkeeping,
+				       redo_parallel::redo_job_queue &a_queue,
+				       const log_rv_redo_context &copy_context)
```

Comment:
Also added `const log_rv_redo_context &copy_context`. Task's redo context has to be similar to the main thread's redo context.